NewSpring/corinth
# chore: Added icons for notifications on Android

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR adds icons for Android notifications.

![Screenshot 2020-06-17 09 11 32](https://user-images.githubusercontent.com/3229463/84903178-c1604800-b07b-11ea-9525-b160aa86383e.png)
![Screenshot 2020-06-17 09 11 24](https://user-images.githubusercontent.com/3229463/84903185-c3c2a200-b07b-11ea-9eeb-009048873d16.png)


### How do I test this PR?
Send yourself a notification from the `alpha-rock` simple communication wizard (https://alpha-rock.newspring.cc/page/1245) while your Android sim is up and running. FWIW, the app should be closed at the time. Otherwise you get that notification that just pops up in the middle of the screen with an OK button and you won't see the icons.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
